aiomisc/utils.py

```diff
@@ -73,3 +73,84 @@ def new_event_loop(pool_size=None) -> asyncio.AbstractEventLoop:
     asyncio.set_event_loop(loop)
 
     return loop
+
+
+def wait_for(*coros, raise_first: bool = True,
+             loop: asyncio.AbstractEventLoop = None):
+
+    tasks = list()
```

Comment:
Better `type: List[asyncio.Task]` here instead of `type: asyncio.Task` in `for`.